c10/core/QScheme.h

```diff
@@ -15,13 +15,15 @@ namespace c10 {
  * Keep this file in sync with torch/nn/_qscheme.py
  */
 enum class QScheme : uint8_t {
-  PER_TENSOR_AFFINE = 0,
-  PER_CHANNEL_AFFINE = 1,
-  PER_TENSOR_SYMMETRIC = 2,
-  PER_CHANNEL_SYMMETRIC = 3,
-  COMPILE_TIME_NUM_QSCHEMES = 4,
+  NO_QUANT = 0,
```

Comment:
where would we use NO_QUANT?

Reply:
hmm, I was thinking that QConfig will need it sometimes, when we do architecture search in the future, we might need to represent no quantization.

Reply:
I think you should represent this with an optional.